modules/ols-about-azure-openai.adoc

```diff
@@ -5,6 +5,4 @@
 [id="ols-about-azure-openai_{context}"]
 = About Azure OpenAI 
 
-To use {azure-openai} with {ols-full}, you must have access to {azure-openai}. If you have already applied for and received access to {azure-openai}, you can ignore the rest of the information in this topic.
-
-{azure-openai} access currently requires an approval process initiated from within the {azure-full} portal. It can take several days for Microsoft to approve your access. When your access to {azure-openai} is approved, make sure you know which Azure subscription your {azure-openai} access is associated with.
+To use {azure-official} with {ols-full}, you must have access to {azure-openai}. If you have already applied for and received access to {azure-openai}, you can ignore the rest of the information in this topic.
```

Comment:
Might be worth checking the naming for "Azure OpenAI" with legal. For example, in serverless docs, you have to use the full name for "Apache Kafka" rather than just "Kafka".

I'm not sure the second sentence here is necessary?
>If you have already applied for and received access to {azure-openai}, you can ignore the rest of the information in this topic.

It comes across as there should be more information in this module following this, but since there isn't (next topic is Watson) maybe it can be left out entirely?

Personally, since the three modules under "Large Language Model (LLM) Requirements" are only about a sentence long each, I wouldn't bother having these as separate modules, I'd just make them subheadings of the `ols-large-language-model-requirements` section.

Also, side note, the title of this assembly should be "Large Language Model (LLM) requirements", since we use sentence case for headings per https://www.ibm.com/docs/en/ibm-style?topic=grammar-capitalization#headings-titles-and-banners